test/sql/cast/string_to_struct_cast.test

```diff
@@ -0,0 +1,463 @@
+# name: test/sql/cast/string_to_struct_cast.test
+# description: cast strings into (nested) list structures
+# group: [cast]
+
+# statement ok
+# PRAGMA enable_verification
```

Comment:
Could you have a look at fixing this? In general all our tests should run with `enable_verification`